EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -1676,6 +1791,51 @@ void V2::World::outputLocalisation() const
 	auto dest = localisationPath + "/text.csv";
 
 	LOG(LogLevel::Info) << "<- Writing Localization Names";
+
+	// Don't copy tag localisation if not needed or overriding mod localisation
+	if (const auto& mod = theConfiguration.getVic2ModName(); !mod.empty())
+	{
+		std::ifstream inNames("./blankMod/output/localisation/0_Names.csv");
+		if (!inNames.is_open())
+			throw std::runtime_error("Could not open blankMod/output/localisation/0_Names.csv for reading");
+		std::ofstream outNames(localisationPath + "/0_Names.csv");
+		if (!outNames.is_open())
+			throw std::runtime_error("Could not open " + localisationPath + "/0_Names.csv for writing");
+
+		// Vic2 tags transformed into corresponding mod tag (e.g for HPM: MLB -> MNP)
+		// MLB is no longer needed, MNP would override mod localisation
+		std::vector<std::string> tagsInMap;
+		for (const auto& tag: countryMapper.getV2TagToModTagMap())
+		{
+			tagsInMap.push_back(tag.first);
+			tagsInMap.push_back(tag.second);
+		}
+		// Vic2 countries assigned new tag (e.g. for HPM: LUN -> LNB)
+		// LUN would override mod localisation for Lundu
+		for (const auto& tag: countryMapper.getReassigningMap())
```

Comment:
so, technically, if you just output updated names into a file that loads before 0_Names there is no need to worry about reassigned countries here. You can leave the regular locs in place.